### DISCUSSION TITLE
Help Pycharm

### DISCUSSION THREAD

**Discussion by KendarKing** [ErkinKraft/Micro-Calc-1.2#12] [Q&A] [answered]
Извините я уже достал наверно, но у меня Pycharm выдаёт ошибку, чёт с python

**Comment by ErkinKraft** [comment, marked as answer]
Ничего страшного, попробуйте следующие:
- Установить python
- Проверить python в переменных средах
- Выбрать python в настройках pycharm
По остальному советую обратиться к гуглу или предоставить саму ошибку
